### DISCUSSION TITLE
Un-fullscreening always starts the animation

### DISCUSSION THREAD

**Discussion by kchibisov** [YaLTeR/niri#83] [General]
Even when nothing changes as a result, just feels like it in place bounces a bit.

**Comment by YaLTeR** [comment]
How can I reproduce the situation where it does an in place bounce?

**Comment by kchibisov** [comment]
It does it always for all windows. So open a window and try to toggle fullscreen. Can open empty workspace, some window, and toggle fullscreen.

**Comment by YaLTeR** [comment]
Converting to a discussion because at the moment it works "as intended". Fullscreen windows don't have struts, so the animation moves the window a tiny bit to the left (gap size => no gap).

The animation could be better, e.g. interpolate the whole window rect to fullscreen, that would make it more obvious. Or it could be disabled specifically when the window is one gap size away from the left. Either way I don't think it's a particularly big deal right now.
